google/cloud/documentai_toolbox/wrappers/page.py

```diff
@@ -16,11 +16,27 @@
 """Wrappers for Document AI Page type."""
 
 import dataclasses
-from typing import List
+import html
+from typing import List, TypeVar, Union
+
+from google.cloud.documentai_toolbox.constants import ElementWithLayout
 
 from google.cloud import documentai
 import pandas as pd
 
+Page = TypeVar("Page")
```

Comment:
Let's not use TypeVar for this.  In the annotations below, you can simply quote the name of the type.